pkg/store/operation/store.go

```diff
@@ -17,24 +17,66 @@ import (
 )
 
 const (
-	nameSpace = "operation"
+	namespace = "operation"
 	index     = "suffix"
 )
 
 var logger = log.New("operation-store")
 
 // New creates new operation store.
 func New(provider storage.Provider) (*Store, error) {
-	store, err := provider.OpenStore(nameSpace)
+	store, err := provider.OpenStore(namespace)
 	if err != nil {
 		return nil, fmt.Errorf("failed to open operation store: %s", err.Error())
 	}
 
+	config, err := provider.GetStoreConfig(namespace)
```

Comment:
The SetStoreConfig logic in the store implementation actually already handles checking for an existing store config. I believe you can simply set the store config here (it won't throw an error).

(Currently CouchDB is the only one with true Store Configuration management, but I plan on keeping the behaviour I described above consistent among store implementations)

Reply:
It throws an error "CouchDB: Internal Server Error: Encountered a conflict while saving the design document.", that's why I did all this error checking.

Reply:
Ah - I will fix this internally. I can update this code later once it's no longer needed.

Reply:
Sounds good